lib/geordi/docker.rb

```diff
@@ -5,6 +5,9 @@
 module Geordi
   class Docker
     DOCKER_COMPOSE_FILE = 'docker-compose.yml'.freeze
+    VNC_DISPLAY = ':17'.freeze
+    VNC_PASSWORD_FILE = File.expand_path('~/.vnc/passwd').freeze # default for "vncpasswd"
```

Comment:
How can this work when you removed the password setting process in _setup_vnc.rb?

Reply:
If someone already used VNC, there is already a passwd file present in the users root directory. It also depends on the authentication method if this is even needed, after some testing it seems like the server (in this case the VNC server running in the docker container), defines the used passwd file at initialization. As far as the docs go, that password file, given it does not change during runtime, does this:
> The passwd-file specifies the file containing the password used to authenticate viewers if VNC authentication is in use. 

Further testing will clear this up I hope.

Reply:
It turns out that the password option was never really used. Though the user can provide the password file, the docker-compose.yml in our legacy docker setup disabled this with `GEORDI_VNC_OPTIONS: '-SecurityTypes None -geometry 1280x1024'`

Using docker with a VNC viewer seems to mitigate security issues. I think this flag was added by @kratob, but I don't know for sure.

I hope this answeres your question.